source/codegen/templates/service_helpers.mako

```diff
@@ -349,16 +248,16 @@ one_of_case_prefix = f'{namespace_prefix}{function_name}Request::{PascalFieldNam
         response->set_${parameter_name}(static_cast<${namespace_prefix}${parameter["enum"]}>(${parameter_name}));
 %  endif
         response->set_${parameter_name}_raw(${parameter_name});
-%elif common_helpers.is_array(parameter['type']):
-%  if handler_helpers.is_string_arg(parameter):
+% elif common_helpers.is_array(parameter['type']):
+%  if service_helpers.is_string_arg(parameter):
```

Comment:
Add a little more space between the `%` and `if`.

Reply:
Along with other comment, adjusted the spaces for these indentation levels to be 1 space for first level and two additional spaces for each subsequent indentation level.